src/Gandi.php

```diff
@@ -38,11 +38,12 @@
     private $apis;
 
     /**
-     * @param FxmlrpcClient $xmlClient
-     * @param string        $apiUrl
-     * @param string        $apiKey
+     * @param string             $apiKey
+     * @param string|null        $apiUrl
+     * @param FxmlrpcClient|null $xmlClient
+     * @param AbstractApi|null   $apis
```

Comment:
BTW, why apis should be passed to the constructor?

It should be internal and instanciated on demand like here: https://github.com/nexylan/graylog-sdk/blob/51aade2fb0185e99464ad42639ddb2dee55c9f89/src/Graylog.php#L136